src/components/Class/ClassList/styles.scss

```diff
@@ -0,0 +1,104 @@
+
+.create-class-button{
+	background: #83B344;
+	margin-left: 17%;
+	width: 10%;
+	height: 40px;
+	border-radius: 15px;
+	color: white;
+	font-size: 17px;
+	font-weight:'bold';
+	border: none ;
+}
+
+
+table { 
+	
+	border-collapse: collapse; 
+	margin:70px auto;
+	box-shadow: 0px 2px 10px 1px rgba(0, 0, 0, 0.3);
+	}
+
+
+tr:nth-of-type(odd) { 
+	// background: #eee; 
+	}
+
+th { 
+	background: #83B344; 
+	color: white; 
+	font-weight: bold; 
+	}
+
+td, th { 
+	padding: 20px; 
+	border: 1px solid #ccc; 
+	text-align: center; 
+	font-size: 19px;
+	font-family: "Times New Roman", Times, serif;
+	}
+
+
+@media 
+only screen and (max-width: 760px),
+(min-device-width: 768px) and (max-device-width: 1024px)  {
+
+	table { 
```

Comment:
Could we fix the indentation? You can take a look in other `styles.scss` files. 🙂

Reply:
done